index.html

```diff
@@ -0,0 +1,229 @@
+                        <div class="title">
+                            <h6 class="tx-custom-color">Topic</h6>
+                        </div>
+                        <div class="desc">
+                            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. </p>
+                        </div>
+                    </div>
+
+                    <div class="program_item">
+                        <div class="img_cover"><img src="assets/images/discourse.png" width="40px" alt="discourse">
+                        </div>
+                        <div class="title">
+                            <h6 class="tx-custom-color">Quiz</h6>
+                        </div>
+                        <div class="desc">
+                            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. </p>
+                        </div>
+                    </div>
+                </div>
+                <div class="join_button_cover">
+                    <button>Join Star Education Society</button>
+                </div>
+            </div>
+        </div>
+    </section>
+    <!-- .......TEACHER SECTION....... -->
+    <section class="teacher_section py-4  d-flex flex-column justify-content-center align-item-center container">
+        <div class="master">
+            <div class="row text-white px-1 py-3 text-center">
+                <h6 class="tx-custom-black">Featured Teachers</h6>
+                <hr class="hr">
+            </div>
+            <div class="master_item" id="a">
+
+            </div>
+            <div class="more">
+                <button id="btn_more"><span>MORE</span><i class="bi bi-chevron-down mx-1"></i></button>
+            </div>
+        </div>
+    </section>
+    <section class="container-fluid bg-custom-black" id="removeableFooter">
+        <div class="container py-5">
+            <div class="row">
+                <div class="col text-center text-white">
+                    <h6>Contact</h6>
+                    <hr class="hr">
+                </div>
+            </div>
+            <div class="row text-center">
+                <div class="col text-white m-1"><a href="#"><img src="assets/icons/mozilla.png" width="80"
+                            alt="mozilla"></a></div>
+                <div class="col text-white m-1"><a href="#"><img src="assets/icons/googlekorea.png" width="80"
+                            alt="google"></a></div>
+                <div class="col text-white m-1"><a href="#"><img src="assets/icons/naver.png" width="80"
+                            alt="naver"></a></div>
+                <div class="col text-white m-1"><a href="#"><img src="assets/icons/daumkakaologo.png" width="80"
+                            alt="daumkakaologo"></a></div>
+                <div class="col text-white m-1"><a href="#"><img src="assets/icons/airbnb.png" width="80"
+                            alt="airbnb"></a></div>
+            </div>
+        </div>
+    </section>
+    <div class="container">
```

Comment:
- [optional] please also use a footer tag for the home page footer section. like the previous reviewer noted This has an advantage when it comes to semantics. Screen readers and SEO needs such information to better understand your page.
- [optional] please kindly increase the size of the footer section as well in both the home and about page of the desktop version. following the original design helps you stick to details